### PR TITLE
Remove redundant symbolic alt to so term hash

### PR DESCRIPTION
**Pull request by nakib103**
Reported in https://github.com/Ensembl/VEP_plugins/issues/710
Also see https://github.com/Ensembl/ensembl-variation/pull/1075

### CODE REVIEW

**Comment by nakib103** [comment]
@nuno-agostinho, 
The unit test on [one before the last](https://github.com/Ensembl/ensembl-vep/pull/1643/commits/ca84f83d6283d573012cb3b1daf5f0543470396e) take accounts the change in ensembl-variation and passed. You can ignore the unit test failure in the last commit.

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]
Same concern as expressed in https://github.com/Ensembl/ensembl-variation/pull/1075#pullrequestreview-2286025167.

Otherwise, I just have small suggestions.

File: modules/Bio/EnsEMBL/VEP/Parser.pm [hunk 1]
```diff
@@ -82,7 +82,11 @@ use FileHandle;
 
 use base qw(Exporter);
 
-our @EXPORT_OK = qw(get_SO_term);
+our @EXPORT_OK = qw(
+    get_SO_term
+    &check_format
+    &_valid_region_regex
```
[the remark this answers is not in the document]
Comment (nakib103) [reply modules/Bio/EnsEMBL/VEP/Parser.pm:88]: it is used in - 
- https://github.com/Ensembl/ensembl-vep/blob/31a3581b84495b617b2f3980da6c6313ca6d238f/modules/Bio/EnsEMBL/VEP/Parser/Region.pm#L171 which is a child class so no need of import.
- `check_format` for `ensembl-variation`. We need to import for that. I am just giving preference to remove redundancy.

**Comment by nakib103** [comment]
Closing this PR and will create a new one.

**Pull request closed without merging**